src/modules/networking/tests/NetworkingTests.cpp

```diff
@@ -218,7 +219,18 @@ namespace OSConfig::Platform::Tests
         "\"Enabled\":\"docker0=true;eth0=false\","
         "\"Connected\":\"docker0=true;eth0=false\"}";
 
-        const char* payloadNmcliNotInstalled =
+        const char* payloadNetworkManagerEnabled =
+        "{\"InterfaceTypes\":\"docker0=bridge;eth0=ethernet\","
+        "\"MacAddresses\":\"docker0=0a:25:3g:6v:2f:89;eth0=00:15:5d:26:cf:89\","
+        "\"IpAddresses\":\"docker0=172.32.233.234,::1;eth0=172.27.181.213,10.1.1.2,fe80::5e42:4bf7:dddd:9b0f\","
+        "\"SubnetMasks\":\"docker0=/8,/128;eth0=/20,/16,/64\","
+        "\"DefaultGateways\":\"docker0=172.17.128.1;eth0=172.13.145.1\","
+        "\"DnsServers\":\"docker0=8.8.8.8,172.29.64.1;eth0=172.29.64.1\","
+        "\"DhcpEnabled\":\"docker0=true;eth0=false\","
+        "\"Enabled\":\"docker0=true;eth0=false\","
+        "\"Connected\":\"docker0=true;eth0=false\"}";
```

Comment:
Move 4 spaces to right (will stop commenting on this, please apply to all multiline string constants)